### DISCUSSION TITLE
Delay in Receiving GitHub Student Developer Pack Benefits

### DISCUSSION THREAD

**Discussion by Yasin-asif** [community/community#142039] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope you're doing well! My GitHub Student Developer Pack application was approved, and I received a confirmation that my benefits would be available soon. However, it has been over 3 days since the approval, and I have not yet received access to my academic benefits.

Could someone kindly assist me with this issue or let me know how much longer I should expect to wait? I understand that delays might occur during busy periods, but any help would be greatly appreciated.

Thank you for your support, and I look forward to hearing back soon.

Best regards,  
Muhammad Yasin

**Comment by davevad93** [comment]
Hi @Yasin-asif ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Yasin-asif was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by mithra0612** [comment]
Typically, access to academic benefits is granted within 7-8 days following approval. However, during peak times, such as the start of a new semester, there may be additional delays.
If it has been over a week and you still haven't received access, I recommend following up with the support team to ensure everything is in order.Keep checking the e-mail account associated with your github application. You'll get an email with subject as [GitHub Education] 💖 @Yasin-asif
